__tests__/app.test.js

```diff
@@ -218,3 +218,66 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {});
+it("returns a staus of 202 and responds with the article", () => {
```

Comment:
This is not quite the right code. I can see why you may have used it but a simple 200 is more appropriate here 👍